src/store/actions.ts

```diff
@@ -853,31 +853,46 @@ export const parseExactNames = (json: { names: [string] }): Array<ConflictListIt
   return names
 }
 
-export const parseRestrictedWords = (resp: RestrictedResponseIF): ParsedRestrictedResponseIF => {
-  const words = resp.restricted_words_conditions
+export const parseRestrictedWords = ({ getters }, resp: RestrictedResponseIF): ParsedRestrictedResponseIF => {
+  const phrases = resp.restricted_words_conditions
   let parsedResp: ParsedRestrictedResponseIF = {
     conditionalInstructions: [],
     conditionalWords: [],
     restrictedWords: []
   }
   // restrictedWords: add any word with cnd_info[..].allow_use = N
   // conditionalWords: all other words in the list
-  for (let i = 0; i < words.length; i++) {
+  for (let i = 0; i < phrases.length; i++) {
+    const phrase = phrases[i].word_info.phrase.toUpperCase()
+
+    // rules for ignoring restricted/conditional phrases
+    const entityCd = getters.getEntityTypeCd
+    if (entityCd === EntityType.CR && Designations[EntityType.CC].words.includes(phrase)) continue
+    if (Designations[entityCd].words.includes(phrase)) continue
```

Comment:
I've never really used `continue` in this context. 
Will this `return` if the condition is NOT met?

Reply:
yeah it just means the loop will go to the next one (so all the code below it will be ignored if the condition is met and the loop will continue)

Reply:
Okie doke!